bundles/org.openhab.binding.hdpowerview/src/test/java/org/openhab/binding/hdpowerview/HDPowerViewJUnitTests.java

```diff
@@ -286,12 +272,62 @@ public void testOnlineCommunication() {
     }
 
     /**
-     * Test generic JSON shades response
+     * Test parsing of ShadePosition.
+     *
+     */
+    @Test
+    public void testShadePositionParsing() {
```

Comment:
I propose to split this method into separate methods, one for each distinct testcase.

Reply:
Ok.